fcl/caf/cafmakerjob_icarus_data_rereco_numi_Run1_ReprocRelease.fcl

```diff
@@ -0,0 +1,28 @@
+#include "cafmakerjob_icarus_data_rereco_numi.fcl"
+
+services.Geometry: @local::icarus_geometry_services_no_overburden_legacy_icarus_v3.Geometry
+
+physics.stream1: [ out1 ]
+
+outputs: {
+   out1: {
+      SelectEvents: [
+         "runprod"
+      ]
+      compressionLevel: 1
+      dataTier: "reconstructed"
+      fileName: "%ifb_%tc-%p.root"
+      module_type: "RootOutput"
+      outputCommands: [
+         "keep *_*_*_*",
+         "drop *_caloskimCalorimetryCryoE_*_*",
+         "drop *_caloskimCalorimetryCryoW_*_*"
+      ]
+      saveMemoryObjectThreshold: 0
+   }
+}
+
+physics.producers.cafmaker.TrackHitFillRREndCut: 9999999
+
+physics.producers.pandoraCaloGausCryoE.T0ModuleLabel: ""
+physics.producers.pandoraCaloGausCryoW.T0ModuleLabel: ""
```

Comment:
Why did you disable it?
From your comments, it sounds like it does not help; does it hurt?

Reply:
I believe was seeing association errors with it. There might be a way around them but since I don't think it is necessary to have, I just gave the empty string.